postgres/Dockerfile

```diff
@@ -1,12 +1,13 @@
-ARG PG_MAJOR
-ARG PGVECTOR_TAG
+ARG PG_MAJOR="14"
+ARG PGVECTOR_TAG="0.8.0"
 
 FROM pgvector/pgvector:${PGVECTOR_TAG}-pg${PG_MAJOR}
 
-ARG PG_MAJOR
-ARG VECTORCHORD_TAG
-ARG PGVECTORS_TAG
-ARG TARGETARCH
+ARG VECTORCHORD_TAG="0.3.0"
+ARG PGVECTORS_TAG="0.3.0"
+ARG TARGETARCH="amd64"
```

Comment:
Putting defaults here breaks things, eg absence of a PGVECTORS_TAG value is meaningful further down. I'd prefer to keep everything explicitly coming from the build matrix. Also PG_MAJOR needs to be there to keep it in scope after the FROM, that'll be what's breaking the build right now.